BackEnd/profiles/tests/test_new_members.py

```diff
@@ -9,21 +9,25 @@
     CategoryFactory,
     RegionFactory,
 )
+from images.factories import ProfileimageFactory
 from profiles.models import Profile
 
 from utils.unittest_helper import utc_datetime
+from utils.completeness_counter import completeness_count
 from utils.dump_response import dump  # noqa
 
 
 class TestCompletenessUpdate(APITestCase):
     def setUp(self) -> None:
-        self.right_image = open(
-            os.path.join(
-                os.getcwd(), "images", "tests", "img", "img_300kb.png"
-            ),
-            "rb",
+        self.banner_path = os.path.join(
+            os.getcwd(), "images", "tests", "img", "img_2mb.png"
```

Comment:
Probably it will be more readable:
```os.path.join( os.getcwd(), "images/tests/img/img_2mb.png")```

Reply:
(not sure if it will work on Windows, if no — return to the curren option).

Reply:
Done